package.json

```diff
@@ -6,7 +6,7 @@
   "publisher": "Imandra",
   "icon": "assets/marketplace.png",
   "galleryBanner": {
-    "color": "#3B3F49",
+    "color": "#221B32",
```

Comment:
Why?

Reply:
Oh I see! I thought we were just using the default marketplace colors, but you're right, these are intended to be extension-specific.